guides/source/active_record_basics.md

```diff
@@ -50,8 +47,7 @@ study them by other means if you would like to learn more.
 
 ### Active Record as an ORM Framework
 
-Active Record gives us several mechanisms, the most important being the ability
-to:
+Active Record gives us the ability to do the following using Ruby objects:
 
 * Represent models and their data.
 * Represent associations between these models.
```

Comment:
Totally optional suggestion but I felt it read better without "these" because we not referring to any specific models. 

```suggestion
* Represent associations between models.
```